press/api/bench.py

```diff
@@ -607,8 +608,7 @@ def deploy_and_update(name, apps_to_ignore=[], sites=[]):
 def create_deploy_candidate(name, apps_to_ignore=[]):
 	rg: ReleaseGroup = frappe.get_doc("Release Group", name)
 	candidate = rg.create_deploy_candidate(apps_to_ignore)
-
-	return candidate
+	return candidate[0]
```

Comment:
This should ideally return all candidates and consumer code should be updated to handle it.